.specify/templates/spec-template.md

```
@@ -0,0 +1,123 @@
+   → If any [NEEDS CLARIFICATION]: WARN "Spec has uncertainties"
+   → If implementation details found: ERROR "Remove technical execution details"
+9. Return SUCCESS when spec is ready for planning
+```
+
+---
+
+## ⚡ Quick Guidelines
+- ✅ Focus on what users experience across Tidigit products
+- ✅ Capture feature gates, personalization, offline/online expectations
+- ✅ Note when the Dexie offline store or DynamoDB sync needs to change
+   - Dexie data model expectations vs DynamoDB sync requirements
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)